app/models/organization.js

```diff
@@ -6,6 +6,10 @@ export default class OrganizationModel extends Model {
   @attr('string') identifier;
 
   @hasMany('contact') contacts;
+  @hasMany('contact') sources;
+  @hasMany('mobile-phone') mobilePhones;
+  @hasMany('telephone') telephones;
+  @hasMany('email-address') emailAddresses;
```

Comment:
Deze relatie noemt in de backend [mail-addresses](https://github.com/kanselarij-vlaanderen/app-persberichten/blob/master/config/resources/contacts.json#L126). In de frontend noemt het model `mail-address`.